src/WebServer.ino

```diff
@@ -3916,17 +3920,21 @@ void handle_control() {
   printToWeb = true;
   printWebString = "";
 
-  if (printToWebJSON)
+  bool unknownCmd = false;
+  if (PluginCall(PLUGIN_WRITE, &TempEvent, webrequest));
+  else if (remoteConfig(&TempEvent, webrequest));
+  else unknownCmd = true;
+
+  if (printToWebJSON) // it is setted in PLUGIN_WRITE (SendStatus)
     TXBuffer.startJsonStream();
   else
     TXBuffer.startStream();
 
-  if (PluginCall(PLUGIN_WRITE, &TempEvent, webrequest));
-  else if (remoteConfig(&TempEvent, webrequest));
+  if (unknownCmd)
+	TXBuffer += F("Unknown or restricted command!");
   else
-    TXBuffer += F("Unknown or restricted command!");
+	TXBuffer += printWebString;
```

Comment:
This is the N'th time I am looking for `printWebString` to see where it is filled.
Maybe we can somehow think of something to make it more logical? (or I should not be so pedantic in checking code ;) )
Not for this PR, just a remark.